UnrealGDKVersion.txt

```diff
@@ -1 +1 @@
-master
\ No newline at end of file
+master
```

Comment:
seems like there's a whitespace diff, but I guess this doesn't really matter anymore with the new CI